src/include/deepstate/DeepState.h

```diff
@@ -48,6 +48,8 @@
 #define assume DeepState_Assume
 #define check DeepState_Check
 
+#define rand DeepState_Int
```

Comment:
I think we want to actually define the symbol, not just macro-ize it.

Reply:
Yeah, though we need access to real rand so it becomes a little annoying for us; feel free to make a better version though. :)

Reply:
Ok, I think we COULD go with this version for now; long-term, we should just make rand implemented as DeepState_Int, and use a PRNG tailored to our needs.